rotkehlchen/db/upgrades/v45_v46.py

```diff
@@ -63,4 +74,58 @@ def _move_extra_data(write_cursor: 'DBCursor') -> None:
         )
         write_cursor.execute('ALTER TABLE evm_events_info DROP COLUMN extra_data;')
 
+    @progress_step(description='Converting asset movements to history events')
+    def move_asset_movements(write_cursor: 'DBCursor') -> None:
+        new_events: list[AssetMovement] = []
+        write_cursor.execute(
+            'SELECT id, location, category, address, transaction_id, timestamp, asset, '
+            'amount, fee_asset, fee, link FROM asset_movements',
+        )
+
+        for row in write_cursor:
+            try:
+                new_events.extend(create_asset_movement_with_fee(
+                    timestamp=ts_sec_to_ms(row[5]),
+                    location=Location.deserialize_from_db(row[1]),
+                    event_type=HistoryEventType.DEPOSIT if row[2] == 'A' else HistoryEventType.WITHDRAWAL,  # noqa: E501
+                    asset=Asset(row[6]),
+                    amount=FVal(row[7]),
+                    fee_asset=Asset(row[8]),
+                    fee=FVal(row[9]),
+                    extra_data=maybe_set_transaction_extra_data(
+                        address=row[3],
+                        transaction_id=row[4],
+                        extra_data={'url': row[10]} if row[10] is not None and row[10] not in ('', 'None') else None,  # in coinbase we were storing 'None' as string.  # noqa: E501
```

Comment:
Note that `link` isn't always a url. Its often an id of some kind. But I guess anything we call this here will not really apply for some of the events, so I suppose using `url` will work.

Reply:
I used url because is what you had in your PR. Intitially I had link that is what we had in the origins structure. Would it make sense to rename it? Maybe something better than link? If changed it needs to be changed also were you used url